File: src/main/java/com/softwareloop/glo/model/Game.java
package com.softwareloop.glo.model;

import lombok.Data;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlAttribute;
import javax.xml.bind.annotation.XmlElement;
import java.util.List;

@Data
@XmlAccessorType(XmlAccessType.FIELD)
public class Game {

    @XmlAttribute
    private String name;

    @XmlAttribute
    private String sourcefile;

    @XmlAttribute
    private String isbios;

    @XmlAttribute
    private String cloneof;

    @XmlAttribute
    private String romof;

    @XmlAttribute
    private String sampleof;

    @XmlAttribute
    private String board;

    @XmlAttribute
    private String rebuildto;

    @XmlAttribute
    private String description;

    @XmlElement(name = "rom")
    private List<Rom> roms;

}

File: src/main/java/com/softwareloop/glo/model/Game.kt
package com.softwareloop.glo.model

import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlAttribute
import javax.xml.bind.annotation.XmlElement

@XmlAccessorType(XmlAccessType.FIELD)
class Game {

    @XmlAttribute
    val name: String? = null

    @XmlAttribute
    val sourcefile: String? = null

    @XmlAttribute
    val isbios: String? = null

    @XmlAttribute
    val cloneof: String? = null

    @XmlAttribute
    val romof: String? = null

    @XmlAttribute
    val sampleof: String? = null

    @XmlAttribute
    val board: String? = null

    @XmlAttribute
    val rebuildto: String? = null

    @XmlAttribute
    val description: String? = null

    @XmlElement(name = "rom")
    val roms: List<Rom>? = null

}